emubd/lfs_emubd.c

```diff
@@ -11,7 +11,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <limits.h>
-#include <dirent.h>
+//#include <dirent.h>
```

Comment:
Are you able to remove this completely? I don't think we need to keep it around.

Reply:
Ah my bad, I pushed a commit that fixes that.